packages/cells/style/collapser.css

```diff
@@ -22,11 +22,29 @@
 
 
 .jp-Collapser-child:hover {
-  background: var(--jp-brand-color1);
+  opacity: .2;
+  border-left: 2px solid var(--jp-brand-color1);
+  border-top: 2px solid var(--jp-brand-color1);
+  border-bottom: 2px solid var(--jp-brand-color1);
 }
 
 
-.jp-Collapser .jp-Collapser-icon {
+.jp-mod-active .jp-Collapser-child:hover {
+  opacity: 1;
+  border: none;
+}
+
+
+.jp-Cell.jp-mod-active .jp-Collapser .jp-Collapser-icon {
   /* Haven't finished the styling, hide for now */
-  display: none;
+  display: block;
```

Comment:
I think the logic here should be this:

* Default state of the icons should be `display: none`;
* Add another rule that sets `display: block` on the icon on `:hover`.
* Add another rule that sets `display: block` on the icon when the cell is selected.